arangod/Aql/OptimizerRules.cpp

```diff
@@ -4505,7 +4506,8 @@ void arangodb::aql::collectInClusterRule(Optimizer* opt, std::unique_ptr<Executi
 
             replaceGatherNodeVariables(plan.get(), gatherNode, replacements);
           } else if (  //! collectNode->groupVariables().empty() &&
-              (!collectNode->hasOutVariable() || collectNode->count())) {
+              (!collectNode->hasOutVariable() || false/*collectNode->count()*/)) {
+            // TODO
```

Comment:
This looks unready yet.
```suggestion
              (!collectNode->hasOutVariable()) {
```